static/src/js/components/AccessMethod/__tests__/AccessMethod.test.js

```diff
@@ -10,12 +10,17 @@ import userEvent from '@testing-library/user-event'
 
 import ResizeObserver from 'resize-observer-polyfill'
 import AccessMethod from '../AccessMethod'
+import AccessMethodRadio from '../../FormFields/AccessMethodRadio/AccessMethodRadio'
 
 beforeEach(() => {
   global.ResizeObserver = ResizeObserver
   jest.clearAllMocks()
 })
 
+jest.mock('../../FormFields/AccessMethodRadio/AccessMethodRadio', () => jest.fn(() => (
```

Comment:
This should allow you to remove all of the repeated calls to `AccessMethodRadio.mockImplementation(...)`

```suggestion
jest.mock('../../FormFields/AccessMethodRadio/AccessMethodRadio', () => jest.fn().mockImplementation(
  jest.requireActual('../../FormFields/AccessMethodRadio/AccessMethodRadio').AccessMethodRadio
))```

Reply:
I see the swodlr mock does this, missed that